olive/passes/pytorch/quantization_aware_training.py

```diff
@@ -86,6 +86,7 @@ def _default_config() -> Dict[str, PassConfigParam]:
             ),
             "gpus": PassConfigParam(type_=int, description="Number of GPUs to use."),
             "seed": PassConfigParam(type_=int, default_value=None, description="Random seed for training."),
+            "checkpoint_path": PassConfigParam(type_=str, description="Path to save checkpoints."),
```

Comment:
if this is None, can the trainer handle it?

Reply:
Yes. The trainer will call os.getcwd to set as current folder. 
Do you suggest to set it as None by default?

Reply:
Updated.

Reply:
I see. thanks for the info. 
I just wanted to make sure the trainer can handle None. The default value here is already None since the default for `default_value` is None